analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/IOperationExtensionsTest.cs

```diff
@@ -59,6 +59,23 @@ public void TrackedSymbol_FieldReference_IsFieldSymbol(string assignment)
             assignmentTarget.TrackedSymbol().Should().Be(fieldReferenceSymbol);
         }
 
+        [DataTestMethod]
+        [DataRow(@"(int i, int j) = (1, 1)")]
```

Comment:
What will `var (i, j)=(1, 1)` do? 🤔

Reply:
It returns a tuple (which TrackedSymbol does not support). I added a test case.